stdlib/planner/group_max_test.flux

```diff
@@ -0,0 +1,113 @@
+import "testing"
+import "testing/expect"
+import "planner"
+
+option now = () => (2030-01-01T00:00:00Z)
+
+input = "
+#datatype,string,long,dateTime:RFC3339,string,string,string,double
+#group,false,false,false,true,true,true,false
+#default,_result,,,,,,
+,result,table,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,system,host.local,load1,1.83
+,,0,2018-05-22T19:53:36Z,system,host.local,load1,1.63
+,,1,2018-05-22T19:53:26Z,system,host.local,load3,1.72
+,,2,2018-05-22T19:53:26Z,system,host.local,load4,1.77
+,,2,2018-05-22T19:53:36Z,system,host.local,load4,1.78
+,,2,2018-05-22T19:53:46Z,system,host.local,load4,1.77
+"
+
+out_max_bare = "
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,double
+#group,false,false,true,true,false,true,true,true,false
+#default,_result,,,,,,,,
+,result,table,_start,_stop,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:46Z,system,host.local,load4,1.77
+"
+
+out_max_window = "
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,double
+#group,false,false,true,true,false,true,true,true,false
+#default,_result,,,,,,,,
+,result,table,_start,_stop,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:26Z,system,host.local,load4,1.77
+"
+
+out_max_agg_window = "
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,double
+#group,false,false,true,true,false,true,true,true,false
+#default,_result,,,,,,,,
+,result,table,_start,_stop,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-23T00:00:00Z,system,host.local,load4,1.77
+"
+
+out_max_agg_window_empty = "
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,double
+#group,false,false,true,true,false,true,true,true,false
+#default,_result,,,,,,,,
+,result,table,_start,_stop,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-23T00:00:00Z,system,host.local,load1,1.83
+,,1,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-23T00:00:00Z,system,host.local,load3,1.72
+,,2,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-23T00:00:00Z,system,host.local,load4,1.78
+"
+
+group_max_bare = () =>
+    testing.loadStorage(csv: input)
+    |> range(start: 2018-05-22T19:53:26Z)
+    |> filter(fn: (r) => r["_value"] == 1.77)
+    |> group(columns: ["_field"])
+    |> max()
+
+group_max_window = () =>
+    testing.loadStorage(csv: input)
+    |> range(start: 2018-05-22T19:53:26Z)
+    |> filter(fn: (r) => r["_value"] == 1.77)
+    |> group(columns: ["_field"])
+    |> window(every: 1d)
+    |> max()
+
+group_max_agg_window = () =>
+    testing.loadStorage(csv: input)
+    |> range(start: 2018-05-22T19:53:26Z)
+    |> group(columns: ["host"])
+    |> aggregateWindow(fn: max, every: 1d)
+
+group_max_agg_window_empty = () =>
```

Comment:
What is the reasoning for having these functions defined outside their testcase block? Are they reused somewhere?